src/components/form/input.tsx

```diff
@@ -17,9 +17,7 @@ interface ReactInput extends React.ComponentPropsWithoutRef<'input'> {
     label_focus_color?: string
     labelSize?: string
     labelTop?: string
-    handleError?: (
-        current_input?: { focus?: () => void } & React.MutableRefObject<HTMLInputElement>,
-    ) => void
+    handleError?: (e: unknown) => void
```

Comment:
using unknown is not the recommended way of giving types. find a proper type.

Reply:
@hirad-rewok yes you are right, but as I checked they don't pass a special argument to this and they are using different arguments so it causes ts errors, I tried to use generic types but it didn't work, so I added `unknown` type here
Do u have any idea for improving this?